README.md

```diff
@@ -85,174 +85,150 @@ We ask that **before you start work on a feature that you would like to contribu
 
 Most contributions require you to agree to a [Contributor License Agreement (CLA)][oss-CLA] declaring that you grant us the rights to use your contribution and that you have permission to do so.
 
-For guidance on developing for PowerToys, please read the [developer docs](/doc/devdocs) for a detailed breakdown. This includes how to setup your computer to compile.
+For guidance on developing for PowerToys, please read the [developer docs](./doc/devdocs) for a detailed breakdown. This includes how to setup your computer to compile.
 
 ## What's Happening
 
 ### PowerToys Roadmap
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.91 - May 2025 Update
+### 0.92 - June 2025 Update
 
-In this release, we focused on new features, stability, and automation.
+In this release, we focused on new features, stability, AOT, and automation.
 
 **✨Highlights**
 
- - We focused on greatly improving the Command Palette's performance and fixing a large number of bugs. Some new features we've added are:
- - Added the ability for Command Palette to search any file using a fallback command.
- - Added the ability to make the Command Palette global hotkey a low-level keyboard hook.
- - Added open URL fallback command for the WebSearch extension, enabling users to directly open URLs in the browser from Command Palette.
- - You can now define custom formats in the Date and Time plugins of PT Run and Command Palette. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Added an option to hide or show the PowerToys system-tray icon. Thanks [@BLM16](https://github.com/BLM16)!  
+ - Major upgrades to Command Palette, including Ahead-of-Time (AOT) readiness, new extension capabilities, and performance/stability fixes across the core UX and extensions.
+ - Mouse button actions for Color Picker so you can choose what left, right, or middle click does. Thanks [@PesBandi](https://github.com/PesBandi)!
```

Comment:
I'd suggest writing style wise for recaps to do "TOOL now has ...." this enables a user to have quick scanning to see what is relevant for them